consensus/polybft/contractsapi/init.go

```diff
@@ -354,6 +354,12 @@ func init() {
 		log.Fatal(err)
 	}
 
+	// Nexus contracts
+	ClaimsTest, ERC1967Proxy, err = initNexusContracts()
```

Comment:
Move this initialization somewhere in e2e test framework. its only part of the test